src/renderer/pages/Settings/Networks/components/NetworkList/NetworkList.tsx

```diff
@@ -1,42 +1,54 @@
 import { useEffect, useState, ReactNode } from 'react';
+import { useUnit } from 'effector-react';
 
 import { ExtendedChain, networkUtils } from '@entities/network';
 import { CaptionText, Counter, Accordion } from '@shared/ui';
+import { filterModel } from '@features/networks/NetworkFilter/model/network-filter';
 
 type Props = {
   title: string;
   isDefaultOpen?: boolean;
-  query?: string;
   networkList: ExtendedChain[];
   children: (network: ExtendedChain) => ReactNode;
 };
 
-export const NetworkList = ({ title, isDefaultOpen, query, networkList, children }: Props) => {
+export const NetworkList = ({ title, isDefaultOpen, networkList, children }: Props) => {
   const [isListOpen, setIsListOpen] = useState(isDefaultOpen);
+  const filterQuery = useUnit(filterModel.$filterQuery);
+
+  console.log('filterQuery', filterQuery);
+  console.log('isDefaultOpen', isDefaultOpen);
+  console.log('isListOpen', isListOpen);
 
   useEffect(() => {
-    if (query) {
-      setIsListOpen(Boolean(query));
+    if (filterQuery) {
```

Comment:
While I've kept the exact same logic, this actually doesn't work and from my tests, it wasn't working before either, as in:
- deactivate "Polkadot"
- make sure the accordion is closed
- type "Polka" in the search
- the Accordion of disabled networks doesn't open


The reason seems to be that the `Disclosure` component from `headlessui` doesn't change the behavior when this props changes.

Reply:
I'll revisit Accordion behaviour for `NetworksList` and let you know

Reply:
The behaviour should be like this:
- disabled section closed by default / active section opened be default
- if filterQuery exists open both sections
- if filterQuery cleared with _cross button_ then collapse disabled section once again
- if filterQuery cleared with `backspace/delete` then disabled section stays opened until user closes it

Reply:
> if filterQuery exists open both sections

This is the problem that I mentioned in my first comment. This is not the case in the `dev` branch, and it's also not the case in this branch either. This could be seen as a bug. However, I looked into it and I don't think there's a non-hacky way to achieve this behavior because the `defaultOpen` prop, is not changing the behavior of the `Disclosure` component. It is a **default** state, and it is only commanded by the button click after the first render.

> if filterQuery cleared with cross button then collapse disabled section once again

There isn't any cross button on Firefox, I wonder if the behavior here is similar to `Ctrl-a` + `Delete`